docs/managers.rst

```diff
@@ -70,6 +70,21 @@ connections and individual per-server:port
 :class:`~urllib3.connectionpool.HTTPSConnectionPool` instances for tunnelled
 HTTPS connections.
 
+Example using proxy authentication:
+
+::
+
+	>>> import base64
+	>>> user = "user"
+	>>> passwd = "123123"
+	>>> headers = {'Proxy-Authorization': 'Basic '+base64.b64encode(user+':'+passwd) }
+	>>> proxy_url = 'http://'+user+':'+passwd+'@localhost:3128'
```

Comment:
I don't think you need the user/passwd in the URL if you're using the `Proxy-Authorization` header?